src/SprykerEco/Zed/UnzerGui/Communication/Controller/DeleteUnzerCredentialsController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerGui\Communication\Controller;
+
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @method \SprykerEco\Zed\UnzerGui\Communication\UnzerGuiCommunicationFactory getFactory()
+ */
+class DeleteUnzerCredentialsController extends AbstractUnzerCredentialsController
+{
+    /**
+     * @var string
+     */
+    public const ROUTE_UNZER_CREDENTIALS_DELETE = '/unzer-gui/delete-unzer-credentials';
```

Comment:
Controllers should not have public constants, please consider moving them to the config, or making them protected both here and in the class that uses it with @uses tag. (### Applicable to **ALL** controllers in the PR.)
```suggestion
    protected const ROUTE_UNZER_CREDENTIALS_DELETE = '/unzer-gui/delete-unzer-credentials';
```